rclcpp/src/rclcpp/typesupport_helpers.cpp

```diff
@@ -139,14 +139,6 @@ get_typesupport_library(const std::string & type, const std::string & typesuppor
   return std::make_shared<rcpputils::SharedLibrary>(library_path);
 }
 
-const rosidl_message_type_support_t * get_typesupport_handle(
```

Comment:
this is not declared or used by anywhere else, i think that is okay to remove this. actually we just forgot to remove this implementation.